bounce/db/user.py

```diff
@@ -17,12 +17,14 @@ class User(Base):
     identifier = Column('id', Integer, primary_key=True)
     full_name = Column('full_name', String, nullable=False)
     username = Column('username', String, nullable=False)
+    secret = Column('secret', String, nullable=False)
     email = Column('email', String, nullable=False)
     created_at = Column(
         'created_at', TIMESTAMP, nullable=False, server_default=func.now())
 
     def to_dict(self):
-        """Returns a dict representation of a User."""
+        """Returns a dict representation of a User.
+        """
```

Comment:
triple quote should be in same line to be consistent with how you did it in other parts of the code